src/core/MOM_isopycnal_slopes.F90

```diff
@@ -202,6 +207,17 @@ subroutine calc_isoneutral_slopes(G, GV, US, h, e, tv, dt_kappa_smooth, use_stan
     endif
   endif
 
+  if ((use_EOS .and. allocated(tv%SpV_avg) .and. (tv%valid_SpV_halo < 1)) .and. &
+      (present_N2_u .or. present(dzSxN) .or. present_N2_v .or. present(dzSyN))) then
```

Comment:
an intermediate `any_present` logical could help here

Reply:
That is a very good suggestion.

Reply:
On the other hand, we generally don't add too many variables that would only be used once, as woudl be the case if we added an `any_present` argument here.  After looking more carefully at the revised code, for now my inclination is to leave it as is, and we can easily refactor in the new temporary variable later if we find that this two-line combination of logical tests is too onerous.